tool/tctl/common/plugin/awsic.go

```diff
@@ -71,6 +72,22 @@ func (a *awsICArgs) validate(ctx context.Context, log *slog.Logger) error {
 	return nil
 }
 
+func (a *awsICArgs) validateSystemCredentialInput() error {
+	if !a.useSystemCredentials {
+		return trace.BadParameter("--useSystemCredentials must be set")
+	}
```

Comment:
Does it mean this flag has to always be set?

Possibly the error message should be:

--use-system-credentials must be set, currently only AWS Local system credentials are supported

Reply:
Done. It now says "--use-system-credentials must be set. The tctl-based AWS IAM Identity Center plugin installation only supports AWS local system credentials"